package.json

```diff
@@ -11,6 +11,8 @@
     "init-db": "node src/init-db.js",
     "create-index": "node scripts/createIndex.js",
     "delete-index": "node scripts/deleteIndex.js",
+    "migrate": "npx sequelize db:migrate",
+    "migrate:undo": "npx sequelize db:migrate:undo",
```

Comment:
I tried to run migration and then undo the migration, all is good.
When I tried to run the migration again and create a resource booking without optional fields and then undo the migration, it failed with error message:
```
ERROR: column "start_date" of relation "resource_bookings" contains null values.
```
@maxceem Do you think it is an issue that should be fixed?

Reply:
@imcaizheng I think this is fine. As we don't have any default values for these fields, we would have to think how to fill the data first, in such a case.